src/W3C-19980720.xml

```diff
@@ -0,0 +1,65 @@
+<SPDX name="W3C Software Notice and License (1998-07-20)" identifier="W3C-19980720" osi-approved="false">
+  <urls>
+    <url>http://www.w3.org/Consortium/Legal/copyright-software-19980720.html</url>
+  </urls>
+  <license>
+    <title>
+      <p>W3C® SOFTWARE NOTICE AND LICENSE</p>
+    </title>
+    <copyright>
+      <p>Copyright (c) 1994-2002 World Wide Web Consortium, (Massachusetts Institute of Technology, Institut
+         National de Recherche en Informatique et en Automatique, Keio University). All Rights Reserved.
+         http://www.w3.org/Consortium/Legal/</p>
+    </copyright>
+    <optional>
+      <p>This W3C work (including software, documents, or other related items) is being provided by the copyright
+         holders under the following license. By obtaining, using and/or copying this work, you (the licensee)
+         agree that you have read, understood, and will comply with the following terms and conditions:</p>
+    </optional>
+    <body>
+      <p>Permission to use, copy, modify, and distribute this software and its documentation, with or without
+         modification, for any purpose and without fee or royalty is hereby granted, provided that you include
+         the following on ALL copies of the software and documentation or portions thereof, including
+         modifications, that you make:</p>
+      <list></list>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>The full text of this NOTICE in a location viewable to users of the redistributed or derivative work.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none
```

Comment:
The body of this license starts at least at line 16 and probably at line 15.  Please check with J.